releasenotes/notes/taper-performance-6da355c04da5b648.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    The :meth:`~qiskit.opflow.Z2Symmetries.taper` has significantly improved
+    performance because intermediate data is simplified.
```

Comment:
Heh, I know I suggested this could be a feature release note, but for a backport we typically only accept bug fixes. Let me rewrite this as a bugfix note and we can slip it in for 0.21.2

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8514/commits/a608ec005310b868e9ccfe57c4519cec958d924c